src/plugins/gopts/README.md

```diff
@@ -6,7 +6,7 @@
 - infos/recommends =
 - infos/placements = procgetstorage
 - infos/status = recommended productive maintained nodep libc
-- infos/metadata = args args/index command opt opt/long opt/arg opt/flagvalue opt/help opt/hidden opt/# opt/#/long opt/#/arg opt/#/flagvalue opt/#/hidden env env/#
+- infos/metadata = args args/index command command/key opt opt/long opt/arg opt/flagvalue opt/help opt/hidden opt/# opt/#/long opt/#/arg opt/#/flagvalue opt/#/hidden env env/#
```

Comment:
Same as in METADATA.ini: The `command/key` metakey is not relevant to the user, so it shouldn't be listed here.

Reply:
Fixed.